.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Test SDK
+on:
+  push:
+    branches: [ 'master', 'feat/**' ]
```

Comment:
todo: rename the branch to main (later)

Reply:
Yes, that is something else I wanted to touch base on.